extensions/amp-analytics/0.1/test/test-variables.js

```diff
@@ -127,56 +129,62 @@ describe('amp-analytics.VariableService', function() {
         new ExpansionOptions({'foo': ' Hello world! '}))
         .then(actual => expect(actual).to.equal('')));
 
-  describe('filter:', () => {
-    const vars = new ExpansionOptions({'foo': ' Hello world! '});
+  describes.fakeWin('filters', {amp: true}, env => {
+    let ampdoc;
+    let urlReplacementService;
 
     beforeEach(() => {
-      sandbox.stub(variables, 'isFilterExperimentOn_').callsFake(() => true);
+      toggleExperiment(env.win, 'url-replacement-v2', true);
```

Comment:
This is not an issue with describes fakeWin, but to be safe, you may want to toggleExperiment to false after each test.

Reply:
done